Source/ablastr/fields/PoissonSolver.H

```diff
@@ -292,8 +302,8 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
                 0,
                 0,
                 1,
-                ng,
-                ng,
+                amrex::IntVect(0),
+                amrex::IntVect(1),
```

Comment:
I noticed that resetting the first argument (corresponding to `src_nghost` in the definition of `ParallelCopy`) to `amrex::IntVect(1)`, as it was before, fixes the "erroneous arithmetic operation" error that was occurring in the tests `ElectrostaticSphereLabFrame_MR_emass_10` and `ElectrostaticSphereEB_RZ_MR`.

What was the reason for deciding to change the code here and pass 0 ghost cells for the source MultiFab, as opposed to 1 ghost cell for the destination MultiFab?

```suggestion
                amrex::IntVect(1),
                amrex::IntVect(1),
```

Reply:
@oshapoval 
Based on the last commit of this PR, I tried changing `1` to `0` in the first of these arguments and I confirm that all four CI tests that are currently passing would crash due erroneous arithmetic operation if we do so. If this turn out to be the one single change that makes the other case work correctly, we would need to find a fix that is good for both that case and all the CI tests here.